.github/workflows/sync-version-with-upstream.yml

```diff
@@ -0,0 +1,40 @@
+name: 🔄 Sync version with upstream
+
+on:
+  # Runs at 10:00 UTC every day
+  schedule:
+    - cron:  '0 10 * * *'
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+
+jobs:
+  sync-version:
+    name: "Update snapcraft.yaml"
+    environment: "Candidate Branch"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.SNAPCRAFTERS_BOT_COMMIT }}
+      - name: Fetch release version
+        run: |
+          VERSION=$(
+            curl -sL https://api.github.com/repos/mattermost/desktop/releases | 
+            jq .  | grep tag_name | grep -v beta | grep -v rc | head -n 1 | cut -d'"' -f4 | tr -d 'v'
+          )
+          sed -i 's/^\(version: \).*$/\1'"$VERSION"'/' snap/snapcraft.yaml
```

Comment:
Wouldn't it be better to also do this with `yq`? Now that you've shown me the light, I want to avoid `sed` :)